windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -173,9 +173,17 @@ $replacement = (Get-Content $replacement_file) -join("`r`n")
 
 
 ##############################
-#  STEP 5 - REMOVE UNNECESSARY FILES.
+#  STEP 4.2 - COPY NECESSARY DLL FILES.
```

Comment:
I'd just make this step 5 and bump the rest of the numbers. The reason why the above step is 4.1 is because it's a temporary hack — the step to copy the DLL is unlikely to go away.

Reply:
SG